### PR TITLE
fix: academic_year mandatory in Student Applicant doctype

### PR DESCRIPTION
**Pull request by RitvikSardana**
Academic Year should be made mandatory in Student Applicant Form.

When a new student applies for a program we can either reject it or accept it.
When We accept it we get an option to “enroll”. As of now this butten when clicked does 2 things

1. Creates a Student which creates a new user with a role of “Student”
2. Goes to program enrollment Page, maps student name, program and enrollment_date but not “academic_year”, due to which we can’t save the Program Enrollment as “academic_year” is a mandatory field in Program Enrollment Doctype.

So we should make “academic_year” field mandatory in Program Enrollment Doctype, so that we can map the academic year as well due to which we will be able to save(create a draft) the program enrollment of that student when we click on “enroll” button.

### CODE REVIEW

File: education/education/web_form/student_applicant/student_applicant.json [hunk 1]
```diff
@@ -359,6 +357,246 @@
...
+   "hidden": 0,
+   "max_length": 0,
+   "max_value": 0,
+   "read_only": 0,
+   "reqd": 0,
+   "show_in_filter": 0
+  },
+  {
+   "allow_read_on_all_link_options": 0,
+   "fieldname": "naming_series",
```
Comment (pateljannat) [inline education/education/web_form/student_applicant/student_applicant.json]: Why do we need naming series in a web form?

**Review by pateljannat — CHANGES_REQUESTED** [review]

File: education/education/web_form/student_applicant/student_applicant.json (hunk 1 above)
Comment (RitvikSardana) [reply education/education/web_form/student_applicant/student_applicant.json]: Fixed it

File: education/education/doctype/instructor/instructor.json [hunk 2]
```diff
@@ -24,7 +24,8 @@
    "fieldtype": "Data",
    "in_global_search": 1,
    "label": "Instructor Name",
-   "reqd": 1
+   "reqd": 1,
+   "unique": 1
```
Comment (pateljannat) [inline education/education/doctype/instructor/instructor.json]: There can be instructors with the same name sometimes.

**Review by pateljannat — CHANGES_REQUESTED** [review]

**Comment by RitvikSardana** [comment]
some fields were automatically added in version 15 in the Student Applicant webform,
I guess the reason was, it took all the fields from the Student Applicant Doctype.


So removed them

**Pull request merged**